v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+# Group Owner
+
+## Owner [/owner]
+Access needs a __private key__.
+
+### List parking spaces [GET /owner/parking-spaces]
+Returns all parking spaces that are owned by the user given by the used private token for the request.
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
```

Comment:
this should be `array[Parking Space]` (also note that it should be uppercase S for Space)

Reply:
Done